src/main/java/minicraft/screen/SkinDisplay.java

```diff
@@ -103,6 +103,9 @@ public SkinDisplay() {
 
     @Override
     public void onExit() {
+		// Achieve Fashion Show:
+		AchievementsDisplay.setAchievement("minicraft.achievement.skin", true);
```

Comment:
Shouldn't this be achieved when opening the menu?

Reply:
Why? If you just open the menu you are only viewing skins not causing an update.
This is also better if you don’t want to change your skin just select the same exact skin.

Reply:
👍🏻